features/user_sessions.feature

```diff
@@ -46,3 +46,66 @@ Feature: User sessions
+    
+  @javascript
+  Scenario: User sees the footer's Our System section
+    When I go to the login page
+    And I should see "Our System"
+    And I should see a link to "https://search.gov/about/updates/releases/" with text "Release Notes"
+    And I should see a link to "https://search.gov/about/updates/roadmap/" with text "Product Roadmap"
+    And I should see a link to "https://search.gov/about/security.html" with text "Security and Compliance"
+    And I should see a link to "https://search.gov/developer/" with text "For Developers"
+    And I should see a link to "https://search.gov/about/policy/tos.html" with text "Terms of Service"
+
+  @javascript
+    Scenario: User sees the footer's Contact Us section
+    When I go to the login page
+    And I should see "Contact Us"
+    And I should see a link to "mailto:search@gsa.gov" with text "Email"
+    And I should see a link to "https://search.gov/tel:(202)-969-7426" with text "Phone"
+
+  @javascript
+    Scenario: User sees the footer's More TTS Services section
+    When I go to the login page
+    And I should see "More TTS Services"
+    And I should see a link to "https://digital.gov/guides/dap/" with text "Digital Analytics Program"
+    And I should see a link to "https://federalist.18f.gov/" with text "Federalist"
+    And I should see a link to "https://cloud.gov/" with text "Cloud.gov"
+    And I should see a link to "https://login.gov/" with text "Login.gov"
+    And I should see a link to "https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions" with text "More TTS Solutions"
+
+    
+  @javascript
+    Scenario: User sees the footer's search.gov section
+    When I go to the login page
+    And I should see "An official website of the GSA's"
+    And I should see a link to "https://gsa.gov/tts" with text "Technology Transformation Services"
+    And I should see a link to "https://www.gsa.gov/about-us" with text "About GSA"
+    And I should see a link to "https://www.gsa.gov/website-information/accessibility-aids" with text "Accessibility support"
+    And I should see a link to "https://www.gsa.gov/reference/freedom-of-information-act-foia" with text "FOIA requests"
+    And I should see a link to "https://www.gsa.gov/reference/civil-rights-programs/notification-and-federal-employee-antidiscrimination-and-retaliation-act-of-2002" with text "No FEAR Act data"
+    And I should see a link to "https://www.gsaig.gov/" with text "Office of the Inspector General"
+    And I should see a link to "https://www.gsa.gov/reference/reports/budget-performance" with text "Performance reports"
+    And I should see a link to "https://www.gsa.gov/website-information/website-policies" with text "Website policies"
+    And I should see "Looking for U.S. government information and services?"
+    And I should see a link to "https://www.usa.gov/" with text "Visit USA.gov"
```

Comment:
I recommend consolidating these into just a couple of scenarios:
```cucumber
Scenario: User sees the Search.gov footer site links
  ...
  
Scenario: User sees the GSA footer links
  ...
```

Reply:
Sounds good, thanks.